MAP/src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import { render, screen } from '@testing-library/react';
```

Comment:
it seems like commenting out a test isn't the best strategy _(even if it isn't useful now)_

Reply:
The only test was the default test case that comes in-built with Create React App. It expects a 'Learn React' link to be rendered on the screen, which of course would not pass. Do still want us to include the test file?

Reply:
yeah, I saw the test you commented out wasn't a good one, but I wonder if there is something else that renders on the screen that could be tested for instead?? maybe if we really are going to remove the test and not add something new, it's better to just delete the file :shrug: